docs/source/tutorials/.gitignore

```diff
@@ -8,4 +8,4 @@ _compiled
 
 *.ipynb
 *.py
-*.rst
```

Comment:
This used to be a safeguard to avoid adding rst files resulting from rendering the qmd file.

Reply:
Right. But it needs to be changed b/c the pre-commit generated `.rst` (which we _do_ want to git track) now live in this directory.